django/db/backends/sqlite3/schema.py

```diff
@@ -70,7 +70,7 @@ def _remake_table(self, model, create_fields=[], delete_fields=[], alter_fields=
         for field in create_fields:
             body[field.name] = field
             # Choose a default and insert it into the copy map
-            if not isinstance(field, ManyToManyField):
+            if field.get_internal_type() != 'ManyToManyField':
```

Comment:
I don't understand; is there a problem with checking the flag on master and the internal type on 1.7?